Tasks/VSMobileCenterTest/using.md

```diff
@@ -0,0 +1,7 @@
+# Using mobile-center test task in VSTS
+
+This task can be used for hosted or [private agents](https://www.visualstudio.com/en-us/docs/build/concepts/agents/agents). 
+
+If you are using this task on in a build definition that will be processed on a private agent, ensure the [mobile-center-cli](https://www.npmjs.com/package/mobile-center-cli).  This package requires node v 6.3.0 or higher.
```

Comment:
Update to "If you are using this task in a build definition that will be run on a private macOS agent, ensure the [mobile-center-cli](https://www.npmjs.com/package/mobile-center-cli) is installed."